components/CategoryImage/CategoryImage.tsx

```diff
@@ -10,21 +10,20 @@ import { type CardSize, getCardImageSizes } from './lib';
 import styles from './CategoryImage.module.scss';
 
 type Props = {
-    category: Category;
-    translatedCategory: TranslatedCategory;
+    image: Category['image'];
+    name: TranslatedCategory['name'];
```

Comment:
Technically this could have been just `ImageDetails` or whatever the type is and `string`, right? 😄